pkg/armhelpers/azurestack/azureclient.go

```diff
@@ -18,6 +18,7 @@ import (
 	"github.com/Azure/azure-sdk-for-go/services/apimanagement/mgmt/2017-03-01/apimanagement"
 	"github.com/Azure/azure-sdk-for-go/services/authorization/mgmt/2015-07-01/authorization"
 	"github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2017-03-30/compute"
+	compute2 "github.com/Azure/azure-sdk-for-go/services/compute/mgmt/2019-07-01/compute"
```

Comment:
@jadarsie Are you O.K. with this hack?

Reply:
`2019-07-01` will likely fail, give me a few to check which version to use

Reply:
`mgmt/2017-03-30/compute` has a `ResourceSkusClient` type, why the need to import `compute2`?

Reply:
but isn't that just for that specific sku generartion command? should not have any impact on how aks-engine works on stack, right?

Reply:
That's right, it's just to support the `aks-engine get-skus` command, which is hidden by default and unlikely to be run on Azure Stack.

I think the problem with the `mgmt/2017-03-30/compute` client was that it has a different func signature for the same call, and bridging the two types was proving difficult. But it's been a while, so I'll dust off this PR and take another look.

Reply:
I don't think it's worth bending over backwards (as I already did) to bridge the two API versions for Azure Stack. I also don't think there's a use case for this hidden, developer-only command in that environment. I suggest we leave it officially unimplemented on Azure Stack and keep the code simpler.